apps/platform/src/render/LinkService.ts

```diff
@@ -35,16 +35,18 @@ export const paramsToEncodedLink = (params: TrackedLinkParts): string => {
 interface TrackedLinkExport {
     user?: User
     campaign?: Campaign
+    userStepId?: number
     redirect: string
 }
 
 export const encodedLinkToParts = async (link: string | URL): Promise<TrackedLinkExport> => {
     const url = link instanceof URL ? link : new URL(link)
     const userId = decodeHashid(url.searchParams.get('u'))
     const campaignId = decodeHashid(url.searchParams.get('c'))
+    const userStepId = decodeHashid(url.searchParams.get('s'))
```

Comment:
Should add this same bit to the `paramsToEncodedLink` method right above so that it is present when possible